rpm/rpm.go

```diff
@@ -199,6 +200,10 @@ func createFilesInsideRPM(info *nfpm.Info, rpm *rpmpack.RPM) error {
 				return err
 			}
 			for src, dst := range globbed {
+				if strings.Contains(src, info.Target) {
```

Comment:
maybe change to `strings.HasSuffix(src, info.Target)`? 

otherwise the target name in the middle of a path would be skipped as well...